documentation/Using-the-SDK/Building-a-table.md

```diff
@@ -1,21 +1,27 @@
 # Building a Table
 
-Table construction happens by interacting with an instance of an `ITableBuilder`. Broadly, `Tables` consist of `TableConfigurations` and Columns. Columns can be further broken down into `ColumnConfigurations` and `Projections`. An `ITableBuilder` has methods to add both Columns and `TableConfigurations`.
+Table construction happens by interacting with an instance of an `ITableBuilder`. Broadly, `Tables` consist of `TableConfigurations` and Columns. Columns can be further broken down into `ColumnConfigurations` and `Projections`. An `ITableBuilder` has methods to add `TableConfigurations` and to establish a row count for the table.
 
-* [Column](#column)
-  * [ColumnConfiguration](#columnconfiguration)
-  * [Projection](#projection)
-  * [Combining ColumnConfiguration and Projections](#combining-columnconfiguration-and-projections)
-* [TableConfiguration](#tableconfiguration)
-  * [ColumnRole](#columnrole)
+*Note*: Every projection in the table must support the table's established row count.
+
+Setting the table's row count (`ITableBuilder.SetRowCount`) returns an instance of `ITableBuilderWithRowCount` which has a method to add columns to the table.
```

Comment:
Maybe this should be its own section below?

Reply:
We had adding columns in this section, and it's part of building a table. I'm ok if we want to make additional changes, but based on the headings, this seemed like the right place to leave it. If you have a specific idea of additional headings, please let me know.